xapian-applications/omega/omindex.cc

```diff
@@ -391,9 +391,6 @@ main(int argc, char **argv)
 
     map<string, string> mime_map;
 
-    index_add_default_filters();
```

Comment:
The default filters need to be set up before the option parsing loop - otherwise the user can't override the default filters on the command line.

Reply:
Sorry, I didn't realize.